tests/functional_tests/ffront_tests/test_math_builtin_execution.py

```diff
@@ -50,13 +49,7 @@ def make_builtin_field_operator(builtin_name: str):
     else:
         raise AssertionError(f"Unknown builtin `{builtin_name}`")
 
-    captured_vars = CapturedVars(
-        nonlocals={"IDim": IDim},
-        globals={builtin_name: getattr(fbuiltins, builtin_name)},
-        annotations=annotations,
-        builtins=set(),
-        unbound=set(),
-    )
+    closure_vars = {"IDim": IDim, builtin_name: getattr(fbuiltins, builtin_name)}
```

Comment:
I was rather annoyed of the amount of code I needed when I wrote this test. This is much easier and cleaner now :-)